pip/pyzmq.file

```diff
@@ -0,0 +1,11 @@
+Requires: libzmq
+
+%define patchsrc \
+cat > setup.cfg <<EOF \
+[global]
+zmq_prefix = $LIBZMQ_ROOT
+
+[build_ext]
+include_dirs = $LIBZMQ_ROOT/include
+library_dirs = $LIBZMQ_ROOT/lib
+EOF
```

Comment:
ah is this change working for you? this looks better

Reply:
I think you are looking at an old commit. The latest one has backslashes at each line, otherwise cmsBuild complains:
```
FATAL: malformed spec found while quering it. Command:
(...)
Resulted in:

error: line 336: Unknown tag: [global]\
error: query of specfile /build/razumov/cmsbuild/w/tmp/tmpspec-py3-pyzmq failed, can't parse
```

Reply:
right, now it looks right